gandalf.js

```diff
@@ -5,66 +5,62 @@
  * for the extras/ plugins if they are loaded into a presentation.
```

Comment:
SCRIPT-The CSS rules here seem to be redundant. Can you please review and remove unnecessary ones?